genon/preprocessor/facade/attachment_processor_guardrail.py

```
@@ -0,0 +1,1626 @@
+# pdf 변환 대상 확장자
+CONVERTIBLE_EXTENSIONS = ['.hwp', '.txt', '.json', '.md', '.ppt', '.pptx', '.docx']
+
+
+
+### 가드레일 용 ###
+import requests
+import re
+import json
+
+GUARDRAIL_WORKFLOW_ID = 694
+GUARDRAIL_BEARER_TOKEN = '23c3898fe3264fd597961af23a68fe7c'
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to: "GUARDRAIL_BEARER_TOKEN"

(S105)